src/networking.c

```diff
@@ -3150,12 +3243,28 @@ NULL
         /* CLIENT SETNAME */
         if (clientSetNameOrReply(c,c->argv[2]) == C_OK)
             addReply(c,shared.ok);
+    } else if (!strcasecmp(c->argv[1]->ptr,"setmeta") && ((c->argc % 2) == 0)) {
+        /* CLIENT SETMETA */
+        if (clientSetMetaOrReply(c) == C_OK)
+            addReply(c,shared.ok);
     } else if (!strcasecmp(c->argv[1]->ptr,"getname") && c->argc == 2) {
         /* CLIENT GETNAME */
         if (c->name)
             addReplyBulk(c,c->name);
         else
             addReplyNull(c);
+    } else if (!strcasecmp(c->argv[1]->ptr,"getmeta")) {
+        /* CLIENT GETMETA */
+        sds meta = NULL;
+        if (c->argc == 2) {
+            meta = addReplyMeta(c, c->meta);
+            addReplyVerbatim(c,meta,sdslen(meta),"txt");
+            sdsfree(meta);
+        } else {
+            meta =addReplyMetaPerFields(c);
```

Comment:
does this "addReply" or just `get` to return the value (i.e. should be named `getClientMetaFields`)?
p.s. missing space.